doc/Extensions/Services.md

```diff
@@ -1,12 +1,144 @@
 source: Extensions/Services.md
 path: blob/master/doc/
-# Setting up Services
+[TOC]
+
+# Setting up services
 
 Services within LibreNMS provides the ability to use Nagios plugins to perform additional monitoring outside of SNMP.
 
 **These services are tied into an existing device so you need at least one device to be able to add it
 to LibreNMS - localhost is a good one. This is needed in order for alerting to work properly.**
 
+## Pre installed plugins
+
+List of pre installed plugins in the format: \<name\> - \<origin\>
+
+Plugins will only load if they are prefixed with "check_" and they have that prefix stripped out when displaying in the "Add Serice" GUI "Type" dropdown list.
+
+* [ajp](https://github.com/bzed/pkg-nagios-plugins-contrib/tree/master/check_ajp) - pkg-nagios-plugins-contrib
```

Comment:
I did mention this on discord but I really don't think we should be maintaing a list like this.

Reply:
> The list of plugins can change, honestly I think just detailing about what they are where people can find them generally is the best option

You did write that. Maybe I interpreted that wrong however I still think a list like this is super useful. I personally have visited this myself several times as a reference since I created it.

If not I guess I could always move this to my own repo specifically for maintaining a list of plugins found in librenms. However I think others would benefit from finding it easily here.

At least my information regarding how plugins are loaded should be considered useful. That isnt documented anywhere to my knowledge so someone trying to script their own would be very confused until they found out it must be prefix with `check_`.

Reply:
That's fine, I won't stand in the way of this being merged but I won't be merging it myself.